jupyterlab_bigquery/src/components/query_editor/query_editor_tab/query_editor_results.tsx

```diff
@@ -59,67 +30,7 @@ interface QueryResultsState {
 interface QueryResultsProps {
   queryResult: QueryResult;
   queryId: QueryId;
-}
-
-interface TablePaginationActionsProps {
-  count: number;
-  page: number;
-  rowsPerPage: number;
-  onChangePage: (
-    event: React.MouseEvent<HTMLButtonElement>,
-    newPage: number
-  ) => void;
-}
-
-function TablePaginationActions(props: TablePaginationActionsProps) {
-  const { count, page, rowsPerPage, onChangePage } = props;
-
-  const handleFirstPageButtonClick = (
-    event: React.MouseEvent<HTMLButtonElement>
-  ) => {
-    onChangePage(event, 0);
-  };
-
-  const handleBackButtonClick = (
-    event: React.MouseEvent<HTMLButtonElement>
-  ) => {
-    onChangePage(event, page - 1);
-  };
-
-  const handleNextButtonClick = (
-    event: React.MouseEvent<HTMLButtonElement>
-  ) => {
-    onChangePage(event, page + 1);
-  };
-
-  const handleLastPageButtonClick = (
-    event: React.MouseEvent<HTMLButtonElement>
-  ) => {
-    onChangePage(event, Math.max(0, Math.ceil(count / rowsPerPage) - 1));
-  };
-
-  return (
-    <div className={localStyles.paginationOptions}>
-      <IconButton onClick={handleFirstPageButtonClick} disabled={page === 0}>
-        <FirstPage />
-      </IconButton>
-      <IconButton onClick={handleBackButtonClick} disabled={page === 0}>
-        <KeyboardArrowLeft />
-      </IconButton>
-      <IconButton
-        onClick={handleNextButtonClick}
-        disabled={page >= Math.ceil(count / rowsPerPage) - 1}
-      >
-        <KeyboardArrowRight />
-      </IconButton>
-      <IconButton
-        onClick={handleLastPageButtonClick}
-        disabled={page >= Math.ceil(count / rowsPerPage) - 1}
-      >
-        <LastPage />
-      </IconButton>
-    </div>
-  );
+  location?: string;
```

Comment:
This prop specifies whether the editor is in the full window or in a notebook cell.

Reply:
Can you turn this into a type? 

Similar to:

```
type FULL_WINDOW = 'FULL_WINDOW';
type IN_CELL = 'IN_CELL'
type QueryEditorType = FULL_WINDOW | IN_CELL;

interface QueryResultsProps {
  queryResult: QueryResult;	  queryResult: QueryResult;
  queryId: QueryId;	  queryId: QueryId;
  editorType?: QueryEditorType;
}
```

Then other locations you can use:

```
if (this.props.editorType === FULL_WINDOW) { ... }
```

Reply:
It seems `this.props.editorType === FULL_WINDOW` won't work because `'IN_CELL' only refers to a type, but is being used as a value here.` Did you mean `this.props.editorType === 'FULL_WINDOW'`?

Is the idea to export the `FULL_WINDOW` and `IN_CELL` types for use elsewhere, or is it to more strongly enforce that we can't pass in any other strings besides these two?

Reply:
Sometimes it is better to refer to const rather than using string. But we can move forward with the strings. That should protect from setting the value to something other than a valid string.